README.md

```diff
@@ -679,19 +679,20 @@ Examples using the test certificates which are self signed for localhost and 127
 ```bash
 > ./gnatsd --tls --tlscert=./test/configs/certs/server-cert.pem --tlskey=./test/configs/certs/server-key.pem
 
-[1844] 2018/02/28 14:31:47.339201 [INF] Starting nats-server version 1.0.6
-[1844] 2018/02/28 14:31:47.339262 [INF] Git commit [not set]
-[1844] 2018/02/28 14:31:47.339410 [INF] Listening for client connections on 0.0.0.0:4222
-[1844] 2018/02/28 14:31:47.339415 [INF] TLS required for client connections
-[1844] 2018/02/28 14:31:47.339418 [INF] Server is ready
+[7531] 2018/03/23 11:46:56.396480 [INF] Starting nats-server version 1.1.0
+[7531] 2018/03/23 11:46:56.396543 [INF] Git commit [not set]
+[7531] 2018/03/23 11:46:56.396725 [INF] Listening for client connections on 0.0.0.0:4222
+[7531] 2018/03/23 11:46:56.396731 [INF] TLS required for client connections
+[7531] 2018/03/23 11:46:56.396743 [INF] Server is ready
 ```
 
 Notice that the log  indicates that the client connections will be required to use TLS.  If you run the server in Debug mode with `-D` or `-DV`, the logs will show the cipher suite selection for each connected client.
 
 ```
-[72167] 2017/06/30 13:08:42.658129 [DBG] ::1:63035 - cid:4 - Starting TLS client connection handshake
-[72167] 2017/06/30 13:08:42.674096 [DBG] ::1:63035 - cid:4 - TLS handshake complete
-[72167] 2017/06/30 13:08:42.674121 [DBG] ::1:63035 - cid:4 - TLS version 1.2, cipher suite TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+[7883] 2018/03/23 11:49:50.603842 [DBG] ::1:58352 - cid:1 - Client connection created
+[7883] 2018/03/23 11:49:50.603908 [DBG] ::1:58352 - cid:1 - Starting TLS client connection handshake
+[7883] 2018/03/23 11:49:50.619115 [DBG] ::1:58352 - cid:1 - TLS handshake complete
+[7883] 2018/03/23 11:49:50.619127 [DBG] ::1:58352 - cid:1 - TLS version 1.2, cipher suite TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
Where is support for 1.3? That coming in golang 1.10?

Reply:
Seems most likely to be included in either Go 1.11 or Go 1.12  https://github.com/golang/go/issues/9671